mod/wikirate_source/set/type_plus_right/source/file/import.rb

```diff
@@ -68,7 +68,11 @@
       end
     end
     if errors.empty?
-      abort :success=>"REDIRECT: #{metric_pointer_card.item_names.first}" 
+      abort success: {
+        name: metric_pointer_card.item_names.first,
+        redirect: true,
+        view: :open
+      }
```

Comment:
this event REALLY needs to be broken down into smaller methods.

(the specific code change is an improvement, though!)